libexec/azhpc-build.sh

```diff
@@ -274,6 +274,7 @@ for resource_name in $(jq -r ".resources | keys | @tsv" $config_file); do
             read_value resource_vm_type ".resources.$resource_name.vm_type"
             read_value resource_image ".resources.$resource_name.image"
             read_value resource_subnet ".resources.$resource_name.subnet"
+            read_value resource_fault_domain_count ".resources.$resource_name.fault_domain_count" 5
```

Comment:
Why is the default 5? Will this change have any impact on HBv1 and HC?

Reply:
The current default in the CLI is 5. So I am making no change to the default.